src/TranslationModel.h

```diff
@@ -1,80 +0,0 @@
-/*
- * TranslationModel.h
- *
- * Main interface for translation API.
- */
-
-#ifndef SRC_TRANSLATOR_TRANSLATIONMODEL_H_
-#define SRC_TRANSLATOR_TRANSLATIONMODEL_H_
-
-#include <future>
-#include <string>
-#include <vector>
-
-// All 3rd party includes
-#include "3rd_party/marian-dev/src/common/options.h"
-
-// All local project includes
-#include "TranslationRequest.h"
-#include "TranslationResult.h"
-#include "translator/definitions.h"
-#include "translator/service.h"
-
-/* A Translation model that translates a plain (without any markups and emojis)
- * UTF-8 encoded text. This implementation supports translation from 1 source
- * language to 1 target language.
- */
```

Comment:
@jerinphilip Could you add this text in the documentation of Service class somewhere? This is an important piece of information that seems to be missing in Service class (correct me if I am wrong) 👍

Reply:
This text is also absorbed.